ci/builder/Dockerfile

```diff
@@ -151,13 +152,13 @@ RUN mkdir rust \
         ;; \
     esac \
     && rm -rf rust \
-    && cargo install --root /usr/local --version "=0.5.2" cargo-about \
-    && cargo install --root /usr/local --version "=1.40.5" cargo-deb \
-    && cargo install --root /usr/local --version "=0.12.2" cargo-deny \
-    && cargo install --root /usr/local --version ="0.9.18" cargo-hakari \
-    && cargo install --root /usr/local --version "=0.9.44" cargo-nextest \
+    && cargo install --root /usr/local --version "=0.5.2" --locked cargo-about \
```

Comment:
Does `--locked` do anything here? There's no Cargo.lock that's pinning the transitive deps, and the top-level version is already pinned.

Reply:
Actually it does do something, weirdly; this failed to compile because a newer version of one of its deps was broken; I think the `--version` with the `--locked` uses the `Cargo.lock` that existed at that version, or something

Reply:
I think so!